test/templates/helpers/templateHelpers.test.ts

```diff
@@ -26,4 +27,30 @@
       expect(getMongoidPrimitive("xhtml")).toEqual("String");
     });
   });
+
```

Comment:
Codacy found an issue: [Some tests seem to be commented](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5738534)